# Available Software Modules

## BEAGLE
**Description:** BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. It can make use of highly-parallel processors such as those in graphics cards (GPUs) found in many PCs.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/BEAGLE/2023.06.06.lua`

---
## gcc
**Description:** The GNU Compiler Collection.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/gcc/12.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/gcc/13.3.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/gcc/14.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/gcc/11.3.lua`

---
## demuxlet
**Description:** Genetic multiplexing of barcoded single cell RNA-seq.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/demuxlet/current.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/demuxlet/default.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/demuxlet/f5044eb.lua`

---
## FASTGA
**Description:** FastGA: Compare two genomes or a genome against itself and output a .1aln, .paf, or .psl file of all alignments found.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/FASTGA/latest.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/FASTGA/1.0.lua`

---
## julia
**Description:** The Julia Programming Language (https://julialang.org/)

- **Module File Path:** `/hpc/modules/modulefiles/Applications/julia/default.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/julia/1.7.3.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/julia/1.8.5.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/julia/1.9.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/julia/1.10.0.lua`

---
## mongosh
**Description:** The MongoDB Shell, mongosh, is a JavaScript and Node.js REPL environment for interacting with MongoDB deployments in Atlas, locally, or on another remote host.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mongosh/2.2.3.lua`

---
## slurm
**Description:** Slurm provides commands and APIs to schedule work on the CZBioHub Cluster Resources.

- **Module File Path:** `/hpc/modules/modulefiles/Core/slurm/current.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Core/slurm/default.lua`

---
## spaceranger
**Description:** Space Ranger is a set of analysis pipelines that process Visium Spatial Gene Expression data with brightfield and fluorescence microscope images.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/spaceranger/2.1.0.lua`

---
## cudnn
**Description:** The NVIDIA CUDA® Deep Neural Network library (cuDNN) is a GPU-accelerated library of primitives for deep neural networks.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/8.7.0.84_cuda10.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/8.8.1.3_cuda11.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/8.8.1.3_cuda12.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/8.9.7.29_cuda11.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/8.9.7.29_cuda12.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cudnn/9.0.0.3_cuda12.lua`

---
## czii
**Description:** Add CZII group managed conda envs and modules.

- **Module File Path:** `/hpc/modules/modulefiles/Groups/czii/default.lua`

---
## BEAST
**Description:** BEAST is a software package for phylogenetic analysis with an emphasis on time-scaled trees.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/BEAST/1.10.4.lua`

---
## nano
**Description:** Gnu Nano

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nano/7.2.lua`

---
## cmake
**Description:** CMake is a powerful, comprehensive solution for managing the software build process.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cmake/3.28.2.lua`

---
## star
**Description:** Spliced Transcripts Alignment to a Reference © Alexander Dobin, 2009-2022. https://www.ncbi.nlm.nih.gov/pubmed/23104886

- **Module File Path:** `/hpc/modules/modulefiles/Applications/star/default.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/star/2.7.10b.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/star/2.7.11b.lua`

---
## raxml-ng-mpi
**Description:** MPI Version: RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/raxml-ng-mpi/1.2.1.lua`

---
## data.science
**Description:** Module for the Data Science group.

- **Module File Path:** `/hpc/modules/modulefiles/Groups/data.science/default.lua`

---
## micro-manager
**Description:** No description available.

- **Module File Path:** `/hpc/user_apps/comp_micro/modulefiles/micro-manager/test.lua`

---
## mustem-gpu
**Description:** Open source version of the MuSTEM multislice electron microscopy simulation code, developed at the University of Melbourne. 

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mustem-gpu/5.3.lua`

---
## caddy
**Description:** Caddy 2 is a powerful, enterprise-ready, open source web server with automatic HTTPS written in Go.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/caddy/2.4.6.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/caddy/2.6.4.lua`

---
## royerlab
**Description:** Module for the Royer Lab.

- **Module File Path:** `/hpc/modules/modulefiles/Groups/royerlab/default.lua`

---
## cellranger-atac
**Description:** Chromium Single Cell ATAC (Assay for Transposase Accessible Chromatin) analyzes thousands of unique open chromatin fragments genome-wide with single cell resolution.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cellranger-atac/2.1.0.lua`

---
## pycharm
**Description:** The Python IDE for Professional Developers (https://www.jetbrains.com/pycharm/)

- **Module File Path:** `/hpc/modules/modulefiles/Applications/pycharm/community-2022.2.lua`

---
## gsl
**Description:** The GNU Scientific Library.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/gsl/2.8-gcc-12.4.lua`

---
## STAR
**Description:** Spliced Transcripts Alignment to a Reference © Alexander Dobin, 2009-2022. https://www.ncbi.nlm.nih.gov/pubmed/23104886

- **Module File Path:** `/hpc/modules/modulefiles/Applications/STAR/default.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/STAR/2.7.10b.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/STAR/2.7.11b.lua`

---
## beagle
**Description:** BEAGLE is a high-performance library that can perform the core calculations at the heart of most Bayesian and Maximum Likelihood phylogenetics packages. It can make use of highly-parallel processors such as those in graphics cards (GPUs) found in many PCs.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/beagle/2023.06.06.lua`

---
## dynamo
**Description:** Dynamo is a software environment for subtomogram averaging of cryo-EM data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/dynamo/1.1.546_MCR-24.1.0_GLNXA64_withMCR.lua`

---
## uv
**Description:** An extremely fast Python package installer and resolver, written in Rust.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/uv/0.2.23.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/uv/0.4.9.lua`

---
## cryosparc
**Description:** CryoSparc

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cryosparc/4.4.0-motioncor2-1.6.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cryosparc/4.5.1-motioncor2-1.6.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cryosparc/4.6.0-motioncor2-1.6.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cryosparc/4.4.1-motioncor2-1.6.4.lua`

---
## fm-index
**Description:** A simple C++ based FM-Index [1] implementation.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/fm-index/1.0.lua`

---
## anaconda
**Description:** Base Anaconda python environment

- **Module File Path:** `/hpc/modules/modulefiles/Applications/anaconda/latest.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/anaconda/2022.05.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/anaconda/2023.03.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/anaconda/2021_09_16.lua`

---
## ruse
**Description:** Ruse: A command line tool to measure process resource usage.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/ruse/2.0.lua`

---
## R
**Description:** R is a free software environment for statistical computing and graphics.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/R/4.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/R/4.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/R/4.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/R/4.3.lua`

---
## emacs
**Description:** Gnu Emacs

- **Module File Path:** `/hpc/modules/modulefiles/Applications/emacs/29.2.lua`

---
## schrodinger
**Description:** Schrodinger Suites is a leading commercial application for computational chemistry. The Biohub has licensed the biologics suite, please contact us if there is interest in purchasing other licenses.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/schrodinger/2021.04.lua`

---
## fastga
**Description:** FastGA: Compare two genomes or a genome against itself and output a .1aln, .paf, or .psl file of all alignments found.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/fastga/latest.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/fastga/1.0.lua`

---
## omnitrace
**Description:** Application Profiling, Tracing, and Analysis from AMD

- **Module File Path:** `/hpc/modules/modulefiles/Applications/omnitrace/1.11.1.lua`

---
## squashfs-tools
**Description:** SQUASHFS - A squashed read-only filesystem for Linux

- **Module File Path:** `/hpc/modules/modulefiles/Applications/squashfs-tools/4.6.1.lua`

---
## cellprofiler
**Description:** CellProfiler is free, open-source software designed to enable biologists without training in computer vision or programming to quantitatively measure phenotypes from thousands of images automatically.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cellprofiler/4.2.1.lua`

---
## matlab
**Description:** MATLAB is a programming and numeric computing platform that should never be used. Please re-write your application, workflows and tools so that they use an open source alternative whenever possible.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/matlab/R2024b.lua`

---
## igv
**Description:** The Integrative Genomics Viewer (IGV) is a high-performance, easy-to-use, interactive tool for the visual exploration of genomic data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/igv/2.17.4.lua`

---
## stui
**Description:** A Slurm dashboard for the terminal.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/stui/0.3.6.lua`

---
## proot
**Description:** chroot, mount --bind, and binfmt_misc without privilege/setup for Linux.

- **Module File Path:** `/hpc/modules/modulefiles/Core/proot/current.lua`

---
## elfindo
**Description:** A parallel find tool for Linux  (https://github.com/breuner/elfindo/ )

- **Module File Path:** `/hpc/modules/modulefiles/Applications/elfindo/0.9.5.lua`

---
## samtools
**Description:** Samtools is a suite of programs for interacting with high-throughput sequencing data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/samtools/1.18.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/samtools/1.19.lua`

---
## ninja
**Description:** Ninja is a small build system with a focus on speed.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/ninja/1.12.1.lua`

---
## chimerax
**Description:** ChimeraX is the next-generation molecular visualization program from the Resource for Biocomputing, Visualization, and Informatics (RBVI), following UCSF Chimera.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/chimerax/1.7.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/chimerax/1.8.lua`

---
## spid
**Description:** This package implements a simple alignment and SNP-calling pipeline for detecting viral and bacterial infectious disease outbreak clusters.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/spid/master.lua`

---
## IGV
**Description:** The Integrative Genomics Viewer (IGV) is a high-performance, easy-to-use, interactive tool for the visual exploration of genomic data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/IGV/2.17.4.lua`

---
## raxml-ng
**Description:** RAxML-NG is a phylogenetic tree inference tool which uses maximum-likelihood (ML) optimality criterion.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/raxml-ng/1.2.1.lua`

---
## BEAST2
**Description:** BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/BEAST2/2.7.4.lua`

---
## mzmine
**Description:** mzmine is an open-source software for mass-spectrometry data processing.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mzmine/4.4.3.lua`

---
## StdEnv
**Description:** No description available.

- **Module File Path:** `/hpc/modules/modulefiles/Core/StdEnv.lua`

---
## rust
**Description:** Rust is blazingly fast and memory-efficient programming language: with no runtime or garbage collector, it can power performance-critical services, run on embedded devices, and easily integrate with other languages. ( https://www.rust-lang.org/ )

- **Module File Path:** `/hpc/modules/modulefiles/Applications/rust/1.77.2.lua`

---
## awscli
**Description:** AWS Command Line Interface (https://docs.aws.amazon.com/cli/index.html)

- **Module File Path:** `/hpc/modules/modulefiles/Applications/awscli/v2.lua`

---
## lightning
**Description:** The deep learning framework to pretrain, finetune and deploy AI models.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/lightning/2.3.1.lua`

---
## dragon
**Description:** Dragon is a composable distributed run-time for managing dynamic processes, memory, and data at scale through high-performance communication objects.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/dragon/0.10.lua`

---
## rsync
**Description:** Rsync is a fast and extraordinarily versatile file copying tool for both remote and local files.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/rsync/default.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/rsync/3.2.7.lua`

---
## beast2
**Description:** BEAST 2 is a cross-platform program for Bayesian phylogenetic analysis of molecular sequences.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/beast2/2.7.4.lua`

---
## ctffind
**Description:** Mindell JA, Grigorieff N. 2003. Accurate determination of local defocus and specimen tilt in electron microscopy.  J Struct Biol. 142:334-47.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/ctffind/4.1.14.lua`

---
## cistem
**Description:** cisTEM

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cistem/b21db55.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cistem/1.0.0.lua`

---
## ffmpeg
**Description:** ffmpeg

- **Module File Path:** `/hpc/modules/modulefiles/Applications/ffmpeg/7.0.lua`

---
## versitygw
**Description:** A High-Performance S3 Translation Service.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/versitygw/1.0.7.lua`

---
## diann
**Description:** a universal automated software suite for DIA proteomics data analysis.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/diann/1.8.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/diann/2.0.1.lua`

---
## r
**Description:** R is a free software environment for statistical computing and graphics.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/r/4.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/r/4.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/r/4.4.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/r/4.3.lua`

---
## papi
**Description:** The Performance Application Programming Interface (PAPI) supplies a consistent interface and methodology for collecting performance counter information from various hardware and software components, including most major CPUs, GPUs, accelerators, interconnects, I/O systems, and power interfaces, as well as virtual cloud environments  ( https://icl.utk.edu/papi/ ).

- **Module File Path:** `/hpc/modules/modulefiles/Applications/papi/7.1.0.lua`

---
## fragpipe
**Description:** FragPipe is a Java Graphical User Interface (GUI) for a suite of computational tools enabling comprehensive analysis of mass spectrometry-based proteomics data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/fragpipe/20.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/fragpipe/21.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/fragpipe/22.0_python3.9.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/fragpipe/22.0.lua`

---
## cuda
**Description:** The NVIDIA® CUDA® Toolkit provides a development environment for creating high performance GPU-accelerated applications.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/10.2.89_440.33.01.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/11.6.2_510.47.03.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/11.7.0_515.43.04.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/11.7.1_515.65.01.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/11.8.0_520.61.05.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.0.0_525.60.13.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.0.1_525.85.12.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.1.1_530.30.02.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.2.0_535.54.03.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.2.1_535.86.10.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.3.2_545.23.08.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.4.0_550.54.14.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.5.1_555.42.06.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.6.3_560.35.05.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.8.0_570.86.10.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cuda/12.6.1_560.35.03.lua`

---
## alphafold
**Description:** AlphaFold is an artificial intelligence (AI) program developed by Alphabet's/Google's DeepMind which performs predictions of protein structure. The program is designed as a deep learning system.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/alphafold/2.1.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/alphafold/2.2.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/alphafold/2.3.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/alphafold/3.0.0.lua`

---
## mafft
**Description:** Multiple alignment program for amino acid or nucleotide sequences

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mafft/7.520.lua`

---
## cellranger
**Description:** Cell Ranger is a set of analysis pipelines that process Chromium single cell data to align reads, generate feature-barcode matrices, perform clustering and other secondary analysis, and more (see list of example workflows and supported libraries).

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cellranger/9.0.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/cellranger/7.0.1.lua`

---
## pixi
**Description:** Pixi is a cross-platform, multi-language package manager and workflow tool built on the foundation of the conda ecosystem.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/pixi/0.25.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/pixi/0.39.5.lua`

---
## mamba
**Description:** Base Mamba python environment

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mamba/23.1.0-3-pypy3.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/mamba/23.1.0-3.lua`

---
## httm
**Description:** httm prints the size, date and corresponding locations of available unique versions (deduplicated by modify time and size) of files residing on snapshots, but can also be used interactively to select and restore files, even snapshot mounts by file. (https://github.com/kimono-koans/httm)

- **Module File Path:** `/hpc/modules/modulefiles/Applications/httm/0.38.0.lua`

---
## beast
**Description:** BEAST is a software package for phylogenetic analysis with an emphasis on time-scaled trees.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/beast/1.10.4.lua`

---
## FastANI
**Description:** FastANI

- **Module File Path:** `/hpc/modules/modulefiles/Applications/FastANI/1.34-gcc-12.4.lua`

---
## relion-turbo
**Description:** RELION.. but turbo (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion-turbo/ver5.0-git-CU80.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion-turbo/ver5.0-git-CU86.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion-turbo/ver5.0-git-CU90.lua`

---
## duc
**Description:** Duc is a collection of tools for inspecting and visualizing disk usage. ( https://duc.zevv.nl/ )

- **Module File Path:** `/hpc/modules/modulefiles/Applications/duc/1.4.4.lua`

---
## bowtie2
**Description:** Bowtie 2 is an ultrafast and memory-efficient tool for aligning sequencing reads to long reference sequences.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/bowtie2/2.1.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/bowtie2/2.5.3.lua`

---
## nextpyp
**Description:** nextPYP

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nextpyp/0.5.lua`

---
## turm
**Description:** TUI for the Slurm Workload Manager 

- **Module File Path:** `/hpc/modules/modulefiles/Applications/turm/0.6.0.lua`

---
## minimap2
**Description:** Minimap2 is a versatile sequence alignment program that aligns DNA or mRNA sequences against a large reference database.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/minimap2/2.26.lua`

---
## comp_micro
**Description:** Module for the computational microscopy group (Shalin Mehta).

- **Module File Path:** `/hpc/modules/modulefiles/Groups/comp_micro/default.lua`

---
## neovim
**Description:** NeoVIM

- **Module File Path:** `/hpc/modules/modulefiles/Applications/neovim/0.10.0.lua`

---
## imod
**Description:** IMOD is a set of image processing, modeling and display programs used for tomographic reconstruction and for 3D reconstruction of EM serial sections and optical sections.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/imod/4.11.25.lua`

---
## nextflow
**Description:** Nextflow is a reactive workflow framework and a programming DSL that eases the writing of data-intensive computational pipelines.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nextflow/22.02.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/nextflow/23.10.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/nextflow/24.10.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/nextflow/current.lua`

---
## FragPipe
**Description:** FragPipe is a Java Graphical User Interface (GUI) for a suite of computational tools enabling comprehensive analysis of mass spectrometry-based proteomics data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/FragPipe/20.0.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/FragPipe/21.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/FragPipe/22.0_python3.9.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/FragPipe/22.0.lua`

---
## fastp
**Description:** A tool designed to provide ultrafast all-in-one preprocessing and quality control for FastQ data.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/fastp/0.24.0.lua`

---
## aria2
**Description:** aria2 is a lightweight multi-protocol & multi-source command-line download utility. It supports HTTP/HTTPS, FTP, SFTP, BitTorrent and Metalink. aria2 can be manipulated via built-in JSON-RPC and XML-RPC interfaces.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/aria2/1.37.0.lua`

---
## lmod
**Description:** Lmod: An Environment Module System

- **Module File Path:** `/hpc/modules/lmod/lmod/modulefiles/Core/lmod.lua`

---
## fpart
**Description:** Fpart is a Filesystem partitioner. It helps you sort file trees and pack them into partitions ( https://www.fpart.org/ )

- **Module File Path:** `/hpc/modules/modulefiles/Applications/fpart/1.6.1.lua`

---
## mpifileutils
**Description:** The mpiFileUtils suite offers MPI-based tools for tasks like copy, remove, and compare for such datasets, delivering orders of magnitude in performance speedup over their single-process counterparts. (https://mpifileutils.readthedocs.io)

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mpifileutils/0.11.1.lua`

---
## axel
**Description:** Axel tries to accelerate the download process by using multiple connections per file, and can also balance the load between different servers.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/axel/2.17.14.lua`

---
## rclone
**Description:** Rclone is a command-line program to manage files on cloud storage.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/rclone/1.65.2.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/rclone/1.67.0.lua`

---
## gurobi
**Description:** Gurobi Optimizer

- **Module File Path:** `/hpc/modules/modulefiles/Applications/gurobi/11.0.2.lua`

---
## meme
**Description:** A set of motif-based sequence analysis tools.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/meme/5.5.5.lua`

---
## zed
**Description:** Zed is a high-performance, multiplayer code editor from the creators of Atom and Tree-sitter. It's also open source.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/zed/0.146.5.lua`

---
## relion
**Description:** RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a software package that employs an empirical Bayesian approach for electron cryo-microscopy (cryo-EM) structure determination.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion/ver5.0-12cf15de-CU80.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion/ver5.0-12cf15de-CU86.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/relion/ver5.0-12cf15de-CU90.lua`

---
## eman
**Description:** EMAN2 in a base Anaconda python environment

- **Module File Path:** `/hpc/modules/modulefiles/Applications/eman/2.99.47_sphire1.4_sparx_huge.lua`

---
## napari
**Description:** Napari containerized with plugins

- **Module File Path:** `/hpc/modules/modulefiles/Applications/napari/test.lua`

---
## ollama
**Description:** Ollam gets you up and running with large language models.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/ollama/0.2.8.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/ollama/0.3.6.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/ollama/0.5.7.lua`

---
## MrBayes
**Description:** MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/MrBayes/git-d5aa8ed.lua`

---
## motioncor3
**Description:** This program corrects whole frame image motion recorded with dose fractionated image stack.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/motioncor3/1.1.1.lua`

---
## openmpi
**Description:** The Open MPI Project is an open source Message Passing Interface implementation that is developed and maintained by a consortium of academic, research, and industry partners.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/openmpi/5.0.3.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/openmpi/4.1.6.lua`

---
## nsight
**Description:** A system-wide performance analysis tool from NVIDIA

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nsight/2024.1.1.59.lua`

---
## cellranger-arc
**Description:** Cell Ranger ARC is a set of analysis pipelines that process Chromium Single Cell Multiome ATAC + Gene Expression sequencing data to generate a variety of analyses pertaining to gene expression (GEX), chromatin accessibility, and their linkage.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/cellranger-arc/2.0.2.lua`

---
## motioncor2
**Description:** This program corrects whole frame image motion recorded with dose fractionated image stack.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/motioncor2/1.6.4.lua`

---
## oracle-java
**Description:** Java is a programming language and computing platform first released by Sun Microsystems in 1995. 

- **Module File Path:** `/hpc/modules/modulefiles/Applications/oracle-java/jdk-17.0.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/oracle-java/jdk-19.0.1.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/oracle-java/jdk-20.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/oracle-java/jdk-17.0.6.lua`

---
## btop
**Description:** Btop

- **Module File Path:** `/hpc/modules/modulefiles/Applications/btop/1.3.0.lua`

---
## resmap
**Description:** ResMap (Resolution Map) is a Python (NumPy/SciPy) application with a Tkinter GUI. It is an easy to use software package for computing the local resolution of 3D density maps studied in structural biology, primarily electron cryo-microscopy (cryo-EM).

- **Module File Path:** `/hpc/modules/modulefiles/Applications/resmap/1.1.4.lua`

---
## hdf5
**Description:** Utilize the HDF5 high performance data software library and file format to manage, process, and store your heterogeneous data. HDF5 is built for fast I/O processing and storage.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/hdf5/1.14.4.lua`

---
## hwloc
**Description:** The Hardware Locality (hwloc) software project aims at easing the process of discovering hardware resources in parallel architectures.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/hwloc/2.10.0.lua`

---
## maxquant
**Description:** MaxQuant is a quantitative proteomics software package designed for analyzing large mass-spectrometric data sets.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/maxquant/2.0.3.1.lua`

---
## spectronaut
**Description:** The gold standard for DIA proteomics analysis

- **Module File Path:** `/hpc/modules/modulefiles/Applications/spectronaut/19.5.241126.62635.lua`

---
## nvhpc-sdk
**Description:** The NVIDIA HPC Software Development Kit (SDK) includes the proven compilers, libraries and software tools essential to maximizing developer productivity and the performance and portability of HPC applications. NOTE: This Module mostly targets the compilers, might have to dig for other features.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nvhpc-sdk/24.3.lua`
- **Module File Path:** `/hpc/modules/modulefiles/Applications/nvhpc-sdk/23.11.lua`

---
## mongodb
**Description:** MongoDB is a general-purpose document database.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mongodb/7.0.8.lua`

---
## settarg
**Description:** The settarg module provides a way to connect the loaded modules with your build system by setting environment variables.


- **Module File Path:** `/hpc/modules/lmod/lmod/modulefiles/Core/settarg.lua`

---
## iqtree
**Description:** Efficient software for phylogenomic inference

- **Module File Path:** `/hpc/modules/modulefiles/Applications/iqtree/2.2.2.6.lua`

---
## jacobo_group
**Description:** Module for the Quantitative Tissue Morphogenesis group (Adrian Jacobo).

- **Module File Path:** `/hpc/modules/modulefiles/Groups/jacobo_group/default.lua`

---
## rstudio
**Description:** Rstudio is an integrated development environment (IDE) for R.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/rstudio/2024.04.1+748.lua`

---
## nvhpc
**Description:** NVIDIA HPC SDK

- **Module File Path:** `/hpc/modules/modulefiles/Applications/nvhpc/2024_2411.lua`

---
## msrsync
**Description:** msrsync (multi-stream rsync) is a python wrapper around rsync

- **Module File Path:** `/hpc/modules/modulefiles/Applications/msrsync/20170730.lua`

---
## hpcx
**Description:** NVIDIA HPC-X toolkit

- **Module File Path:** `/hpc/modules/modulefiles/Applications/hpcx/2.19.lua`

---
## lsgres
**Description:** No description available.

- **Module File Path:** `/hpc/user_apps/comp_micro/modulefiles/lsgres/test.lua`

---
## mrbayes
**Description:** MrBayes is a program for Bayesian inference and model choice across a wide range of phylogenetic and evolutionary models. MrBayes uses Markov chain Monte Carlo (MCMC) methods to estimate the posterior distribution of model parameters.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/mrbayes/git-d5aa8ed.lua`

---
## promtail
**Description:** Promtail is an agent which ships the contents of local logs to a private Grafana Loki instance or Grafana Cloud.

- **Module File Path:** `/hpc/modules/modulefiles/Applications/promtail/3.3.2.lua`

---
